### ISSUE TITLE
Rawspeed update

### ISSUE THREAD

**Issue by esbante** [rawstudio/rawstudio#44]
Would it be possible to update to the latest rawspeed version for better camera support?

**Comment by sergiomb2** [comment]
not easy since rawspeed change is structure a lot , rawspeed has moved to https://github.com/darktable-org/rawspeed 
and may help see https://github.com/klauspost/rawspeed/issues/171

**Comment by esbante** [comment]
That's too bad. Is there an easier way to add support for newer cameras like for example Sony a7r3?

**Comment by sergiomb2** [comment]
@esbante shouldn't be that hard , rawspeed now can be build apart [1]  , load that library , we just need change the load-rawspeed if API change , and we can check that on darktable itself (I guess)



[1] https://build.opensuse.org/package/view_file/graphics:darktable:master/rawspeed/rawspeed.spec?expand=1

**Comment by sergiomb2** [comment]
Please I'd like that you test my 3 new commits [1] , specially "readd load-dcraw updated to ufraw-0.22"
if you have feedback please write in issues of the project [2]
[1] https://github.com/sergiomb2/rawstudio/commits/devel
[2] https://github.com/sergiomb2/rawstudio/issues

**Comment by sergiomb2** [comment]
JFTR rawspeed is currently maintain on https://github.com/darktable-org/rawspeed and here we have an build of rawspeed as an independent package here (1) . 
BTW I also checked libRaw and libRaw have the code of ufraw (2)

(1)
https://build.opensuse.org/package/show/graphics:darktable:master/rawspeed

(2)
https://github.com/LibRaw/LibRaw/commit/d1975cb0e055d2bfe58c9d845c9a3e57c346a2f9

**Comment by LebedevRI** [comment]
> JFTR rawspeed is currently maintain on https://github.com/darktable-org/rawspeed

> and here we have an build of rawspeed as an independent package here (1) .
Not really. This is only a sudo-ci. It does not and will not produce an useable shared library.

> BTW I also checked libRaw and libRaw have the code of ufraw (2)
> 
> (1)
> https://build.opensuse.org/package/show/graphics:darktable:master/rawspeed
> 
> (2)
> [LibRaw/LibRaw@d1975cb](https://github.com/LibRaw/LibRaw/commit/d1975cb0e055d2bfe58c9d845c9a3e57c346a2f9)
